cloud-functions/functions/reset-infrastructure.js

```diff
@@ -28,244 +28,140 @@ async function addMinimalSampleData() {
-                      await db
-                        .collection('users')
-                        .doc('disabled@soylentgreen.com')
-                        .set({
-                          isAdmin: false,
-                          organizationID: soylentGreenID,
-                          disabled: true,
-                          firstName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
-                          lastName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
-                          isFirstTimeUser: false,
-                        });
-                      await db
-                        .collection('users')
-                        .doc('disabled@initech.com')
-                        .set({
-                          isAdmin: false,
-                          organizationID: initechID,
-                          disabled: true,
-                          firstName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
-                          lastName: uniqueNamesGenerator({ dictionaries: [names], length: 1 }),
-                          isFirstTimeUser: false,
-                        });
-                      await auth.createUser({
-                        email: 'disabled@soylentgreen.com',
-                        password: 'disabled@soylentgreen.com',
-                      });
-                      console.log(
-                        `Successfully created Soylent Green disabled user with username/password disabled@soylentgreen.com`
-                      );
-                      await auth.createUser({
-                        email: 'disabled@initech.com',
-                        password: 'disabled@initech.com',
-                      });
-                      console.log(
-                        `Successfully created Initech disabled user with username/password disabled@initech.com`
-                      );
-                    } catch (err) {
-                      throw err;
-                    }
-                  } catch (err_1) {
-                    throw err_1;
-                  }
-                } catch (err_2) {
-                  throw err_2;
-                }
-              } catch (err_3) {
-                throw err_3;
-              }
-            } catch (err_4) {
-              throw err_4;
-            }
-          } catch (err_5) {
-            throw err_5;
-          }
-        } catch (err_6) {
-          throw err_6;
-        }
-      } catch (err_7) {
-        throw err_7;
-      }
-    } catch (err_8) {
-      throw err_8;
-    }
-  } catch (err_9) {
-    throw err_9;
-  }
+async function createUser(user) {
+  await db.collection('users').doc(user.email).set({
+    isAdmin: user.isAdmin,
+    organizationID: user.organizationID,
+    disabled: user.disabled,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    isFirstTimeUser: user.isFirstTimeUser,
+  });
+  await db.collection('userImages').doc(user.email).set({});
```

Comment:
Should probably be consistent with the change you made in `cloud-functions/functions/src/index.ts`:

```suggestion
  await db.collection('userImages').doc(user.email).set({ imageBlob: null });
```